src/EditorFeatures/VisualBasicTest/AddAccessibilityModifiers/AddAccessibilityModifiersTests.vb

```diff
@@ -202,5 +212,195 @@ namespace N
     end namespace
 end namespace")
         End Function
+
+                <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddAccessibilityModifiers)>
+        Public Async Function TestAllConstructsWithOmit() As Task
+            Await TestInRegularAndScriptAsync(
+"
+namespace N
+    namespace Outer.Inner
+        Friend class {|FixAllInDocument:C|}
+            Public class NestedClass
+            end class
+
+            Public structure NestedStruct
+            end structure
+
+            Private f1 as integer
+            Private f2, f3 as integer
+            Private f4, f5 as integer, f6, f7 as boolean
```

Comment:
can you test consts and shared fields as well?  i want to make sure that the new modifier logic doesn't screw them up.

Reply:
(basically, i didn't care before, since we were adding modifiers) .  but now that we're removing modifiers, i want to make sure that doesn't negatively affect any other existing modifiers that may be there.

Reply:
Done, added tests for const and shared with private and public modifiers.